File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task3-WebClient/src/app/core/models/pet/pet-notification-profile.ts
import { Guid } from "guid-typescript";
import { PetType } from "../../enums/pet-type";
import { DiaryNoteListItem } from "../diary-note/diary-note-list-item";

export interface PetNotificationProfile {
	petId: Guid;
	nickName: string;
	petType: PetType;
	breed: string;
	characteristics: string;
	illnesses: string;
	preferences: string;
	documents: DiaryNoteListItem[];
}

File: apzkr-pzpi-21-5-vyshniak-viktoriia/Task1-Server/BLL/Infrastructure/Models/Pet/PetNotificationProfile.cs
using BLL.Infrastructure.Models.DiaryNote;
using Common.Enums;

namespace BLL.Infrastructure.Models.Pet;
public class PetNotificationProfile
{
    public Guid PetId { get; set; }

    public string NickName { get; set; }

    public PetType PetType { get; set; }

    public string Breed { get; set; }

    public string Characteristics { get; set; }

    public string Illnesses { get; set; }

    public string Preferences { get; set; }

    public IEnumerable<DiaryNoteListItem> Documents { get; set; }
}
